src/sentry/static/sentry/app/components/versionHoverCard.tsx

```diff
@@ -16,15 +15,26 @@ import withApi from 'app/utils/withApi';
 import Clipboard from 'app/components/clipboard';
 import {IconCopy} from 'app/icons';
 import Version from 'app/components/version';
+import {Client} from 'app/api';
+import {Release, Deploy} from 'app/types';
 
-class VersionHoverCard extends React.Component {
-  static propTypes = {
-    api: PropTypes.object,
-    version: PropTypes.string.isRequired,
-    orgId: PropTypes.string.isRequired,
-    projectId: PropTypes.string.isRequired,
-  };
+type Props = {
+  api: Client;
+  orgSlug: string;
+  projectSlug: string;
+  releaseVersion: string;
+};
+type State = {
+  loading: boolean;
+  error: boolean;
+  data: {};
```

Comment:
Is data used?

Reply:
This component is being reworked in another PR and `data` is being deprecated.